android/gradle.properties

```diff
@@ -17,3 +17,10 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemor
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 RNGH_kotlinVersion=1.6.21
+
+# Path to codegen output directory with this library view managers' interfaces & delegates. Used by `copyCodegenArtifacts` task that helps to synchronize newly generated files with their Paper counterparts.
+codegenArtifactsSource=android/build/generated/source/codegen/java/com/facebook/react/viewmanagers
+
+# Path to directory with view managers' interfaces & delegates used while running on Paper architecture. This property is used as the output path for `copyCodegenArtifacts` task.
```

Comment:
I believe that this should be `managers`.
Also, what's the difference between the first and second property? I can't spot it 🤔 

```suggestion
# Path to codegen output directory with generated view managers, interfaces and delegates of this library.
# Used by `copyCodegenArtifacts` task that helps to synchronize newly generated files with their Paper counterparts.
codegenArtifactsSource=android/build/generated/source/codegen/java/com/facebook/react/viewmanagers

# Path to directory with view managers, interfaces and delegates used while running on Paper architecture.
```

Reply:
View managers aren't generated only interfaces and delegates. Those interfaces and delegates are (essentially) of a specific view manager, hence, they are possessive here.

And the difference between them is the one points to the build folder (where the specs are generated) and the other to the directory inside the package (where files for old arch compatibility should be).

Reply:
@j-piasecki I see, you're right 👍